examples/ct/structures_test.go

```diff
@@ -3,11 +3,21 @@ package ct
 import (
 	"encoding/base64"
 	"testing"
+	"time"
 
+	"crypto/sha256"
+	"github.com/google/certificate-transparency/go"
+	"github.com/google/certificate-transparency/go/fixchain"
+	"github.com/google/trillian"
 	"github.com/google/trillian/crypto"
+	"github.com/google/trillian/examples/ct/testonly"
 	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/mock"
+	"io"
 )
 
+var fixedTime time.Time = time.Date(2016, 21, 7, 12, 15, 23, 0, time.UTC)
```

Comment:
Yes. I could parse the time from a string but that can fail. It's just the date and time I wrote the code.